.woodpecker.yml

```
@@ -9,7 +9,9 @@ steps:
   check_ansible_format:
     image: alpine:3
     commands:
-      - apk add ansible
+      - apk add pipx
+      - pipx install --include-deps ansible
+      - export PATH=/root/.local/bin:$${PATH}
```

```suggestion
      - export PATH=/root/.local/bin:$${PATH}
      - apk add pipx
      - pipx install --include-deps ansible
```

this just gets rid of the warning during `pipx install` about the bin path not being in `$PATH`, but it doesn't really affect the outcome.

---

Good idea. I had considered this at some point, but then didn't get to it.